.github/workflows/ci.yml

```diff
@@ -125,6 +129,10 @@ jobs:
           path: bin/libgodot-cpp.osx.release.64.a
           if-no-files-found: error
 
+      - name: Build test GDNative library
+        run: |
+          scons target=release platform=osx bits=64 -j $(sysctl -n hw.logicalcpu) -C test;
```

Comment:
Is `;` necessary?

Reply:
No, I just copied it from previous travis yml

Reply:
Fixed.